### DISCUSSION TITLE
Segmentation fault

### DISCUSSION THREAD

**Discussion by albenstein** [su2code/SU2#2116] [General]
### Discussed in https://github.com/su2code/SU2/discussions/2101

<div type='discussions-op-text'>

<sup>Originally posted by **albenstein** July 26, 2023</sup>
*If this is a general question or difficulty building/using the code, please open a "Discussion" instead.*

**Describe the bug**
*Give a clear and concise description of what the bug is and what you expect the behavior to be instead. Please provide enough instructions and data for a developer to reproduce the problem (config, mesh, commands, etc.). Please provide the smallest working example that reproduces the problem.*

**Bug report checklist**
*Please make sure that you have followed the checklist below, many common problems can be solved by:*

- [x] Consulting the build instructions (https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/).
- [x] Looking for similar problems on GitHub or CFD-Online (https://www.cfd-online.com/Forums/su2/).
- [x] Updating to the newest version of SU2 (either develop, master, or the pre-built executables https://su2code.github.io/download.html).

**Desktop (please complete the following information):**
 - OS: [e.g., Linux (Redhat 7.9)]
 - C++ compiler and version: [e.g., g++ (GCC) 8.5.0]
 - MPI implementation and version: [e.g., OpenMPI 4.0.3]
 - SU2 Version: [e.g., v7.5.1]

When I use SU2 with QuickStart config file with OpenMPI, it gives error like in attached file.
Command is:
`mpirun -np 8 --allow-run-as-root SU2_CFD inv_NACA0012.cfg`

[segmentation_fault.txt](https://github.com/su2code/SU2/files/12170716/segmentation_fault.txt)
</div>

**Comment by albenstein** [comment]
I've also asked on OpenMPI mail-list and they said "Can you or they provide a small, simple MPI application that replicates the issue?  That would be something we could dig into and investigate."

**Comment by pcarruscag** [comment]
So the solver runs ok but then crashes when it get to writing the output files?
Can you try
VOLUME_OUTPUT= RESTART_ASCII
Just to isolate the problem

**Comment by albenstein** [comment]
@pcarruscag It crashes when it get to writing the output files. Where can I try VOLUME_OUTPUT, in config file?

**Comment by pcarruscag** [comment]
Yes in the config, it's OUTPUT_FILES= RESTART_ASCII sorry

**Comment by albenstein** [comment]
I've just run with your command, output is: 
[SU2_restart_ascii.txt](https://github.com/su2code/SU2/files/12358842/SU2_restart_ascii.txt)

**Comment by pcarruscag** [comment]
Ok so the other parts of MPI are working fine but file operations are having some issue.
You can remove the OUTPUT_FILES line from the config.
When you run SU2, are you running in a clean directory? Does it segfault in a clean directory? Or only once the files already exist?
I also see you are running on the head node. Have you tried running SU2 on a compute node of the cluster?

**Comment by albenstein** [comment]
@pcarruscag Sorry for late reply.
I was running SU2 on dirty directory. I tried to run on clean directory, it did not segfault in a clean directory.
I also tested on compute node and there was no segfault error on clean directory.
